user-service/src/main/java/rs/edu/raf/banka1/bootstrap/BootstrapData.java

```diff
@@ -91,17 +95,30 @@ public void run(String... args) throws Exception {
         customer.setActive(true);
         customerRepository.save(customer);
 
+        Card card = new Card();
+        card.setCardName("VISA");
+        card.setCardLimit(900);
+        card.setCardNumber("11111");
+
```

Comment:
Ove kartice se isključivo prave u okviru account-a za sada, tako da ne bi trebalo da nam vise kartice ovako.

Reply:
kako mislis to da se prave u okviru account-a? 
BankAccount bankAcc1 = new BankAcc();
Card card = new Card()
card.setAccountNumber(bankAccount1.getAccountNumber()); ovako su mi povezane

Reply:
**saveBankAccount** pravi 2 kartice (pored toga što čuva `bankAccount`)